### DISCUSSION TITLE
Can't upload proof of student identification

### DISCUSSION THREAD

**Discussion by C-Nuteras** [community/community#129478] [GitHub Education]
### Hi everyone,

Hi,

I've been trying to upload a photo of my student ID to Guthub to access GitHub education. The photo requirements are inconsistent. I have extensively resized the image to be within the ratios and within the 100kb-1mb size limit. My photo is still denied. Can I please submit my proof to an email address? I cannot upload documents to prove my student status as the upload section only allows images. These documents are no longer sent in hard copy mail, they are paperless. Can I please have some assistance? Many other students are facing the same issue...

My image was originally to a high standard and was greater than 1MB, I resized it to fit the standards and it is now within the standards but still being rejected. 
Thank you

**Comment by VOID404** [comment]
I had to compress the image below 500K. With anything larger (but below 1M) I had a "Payload to large" error from the API. The error wasn't displayed anywhere in the UI, I had to open dev tools, enable keeping logs and then I was able to debug the problem.
